en-US/emails/2017/fx_mobile_update.lang

```diff
@@ -0,0 +1,48 @@
+## NOTE: Email is about the Pocket feature built in to Firefox.
+## NOTE: Pocket as a brand remains in English.
+## NOTE: Bug 1400968; Polish only
+
+
+# Subject Line + Preheader
+;What’s new with Firefox mobile
+What’s new with Firefox mobile
+
+# Headline
+;Our goal is simple: make Firefox Mobile as awesome as possible, and make sure you know all the awesome things you can do with it. Look out for the Firefox Mobile Update in your in-box whenever we have exciting stuff to share.
+Our goal is simple: make Firefox Mobile as awesome as possible, and make sure you know all the awesome things you can do with it. Look out for the Firefox Mobile Update in your in-box whenever we have exciting stuff to share.
+
+# Introduction
+;Sure, you COULD have a million tabs open or an inbox full of articles you emailed yourself. Or you could just click Pocket.
```

Comment:
Where is this string coming from? The headline from the spreadsheet is missing.

Reply:
argh - jump the tab to a different email!